examples/translation/train.py

```diff
@@ -44,13 +29,13 @@ def main(fast_dev_run=True):
     train = "train" if not fast_dev_run else "validation"
 
     logger.info("build vocabs")
-    source_text_transform = WhitespaceTokenizer(trainable=True)
-    target_text_transform = WhitespaceTokenizer(trainable=True)
-    source_vocab = build_vocab(wmt14["cs"][train], source_text_transform)
-    target_vocab = build_vocab(wmt14["en"][train], target_text_transform)
-    source_text_transform.trainable = False
-    target_text_transform.trainable = False
-
+    source_text_transform = WhitespaceTokenizer()
+    source_text_transform.build_vocab(map(lambda x:x['text'],wmt14["cs"][train]))
+    target_text_transform = WhitespaceTokenizer()
+    target_text_transform.build_vocab(map(lambda x:x['text'],wmt14["en"][train]))
+    source_vocab = source_text_transform.vocab.get_stoi()
```

Comment:
Why use the stoi dictionary instead of the full class as it was before?

Reply:
> Why use the stoi dictionary instead of the full class as it was before?

This change is compatible with previous version. get_stoi() on Vocab returns python dict which was also the case earlier https://github.com/hudeven/text/blob/9fc5b4962282d7e1dc8d5fa5eb3258f253981060/examples/translation/train.py#L27